SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -913,6 +936,14 @@ void USpatialNetDriver::BeginDestroy()
 			}
 		}
 
+		if (RoutingSystem)
+		{
+			RoutingSystem->Destroy(Connection);
+
+			Connection->Flush();
+			FPlatformProcess::Sleep(0.1f);
```

Comment:
What are these sleeps for?

Reply:
Don't know, I reproduced the pattern, just in case.